forest/jasmine/traj2stats.py

```diff
@@ -9,6 +9,7 @@
 import pickle
 from typing import Dict, List, Optional, Tuple
 
+from datetime import datetime
```

Comment:
The `datetime` module is part of the standard library, so should be in the block above
https://docs.python.org/3.8/library/datetime.html